website/static/js/fangorn.js

```diff
@@ -12,6 +12,8 @@ var tbOptions;
 
 var tempCounter = 1;
 
+var uploadInProgress = false;
```

Comment:
Can `uploadInProgress` be an attribute of a `Fangorn` instance instead of a global? Best to avoid globals where possible.